logicrunner/executionbroker.go

```diff
@@ -290,22 +290,36 @@ func (q *ExecutionBroker) IsKnownRequest(ctx context.Context, req insolar.Refere
 
 // startProcessors starts independent processing of mutable and immutable queues
 func (q *ExecutionBroker) startProcessors(ctx context.Context) {
-	q.startProcessor(ctx, &q.immutable)
+	for i := q.startProcessorCount(ctx, &q.immutable); i > 0; i-- {
```

Comment:
I don't like this code. startProcessors is called all the time concurrently and old code was guaranteeing that processor will be started only one. New code starts fights around counting queue length, available slots left. Also, this code gets complicated and I feel like it is possible that we wouldn't start a processor when queue is not empty.